docs/install/android.mdx

```diff
@@ -8,63 +8,81 @@ title: Build for Android
+| **`-DPCAP_INCLUDE_DIR=<LIBPCAP_INCLUDE_DIR>`**     | The path of lipbpcap include files, for example: `${LIBPCAP_DIR}/include/`. This is a mandatory option                                                                                                                                                                      |
+| **`-DPCAPPP_BUILD_EXAMPLES=<ON/OFF>`**             | Build PcapPlusPlus examples, in most cases should be set to `OFF`. If you want to run [PcapPlusPlus examples](../examples) on an Android device you need shell access, and for some of them you also need a rooted device. The examples will reside in `build/examples_bin` |
+| **`-DPCAPPP_BUILD_TESTS=<ON/OFF>`**                | Build PcapPlusPlus tests, in most cases should be set to `OFF`                                                                                                                                                                                                              |
+| **`-DPCAPPP_BUILD_TUTORIALS=<ON/OFF>`**            | Build PcapPlusPlus tutorials. This option is only available if `DPCAPPP_BUILD_EXAMPLES=ON` (default value is `OFF`)                                                                                                                                                         |
+| **`-DPCAPPP_INSTALL=<ON/OFF>`**                    | Install PcapPlusPlus (default value is `ON` if building the project itself, otherwise `OFF`)                                                                                                                                                                                |
+| **`-DBUILD_SHARED_LIBS=<ON/OFF>`**                 | Build shared libs (default value is `OFF`)                                                                                                                                                                                                                                  |
+| **`-DPCAPPP_ENABLE_PCAP_IMMEDIATE_MODE=<ON/OFF>`** | Enable pcap immediate mode (default value is `OFF`)                                                                                                                                                                                                                         |
+| **`-DPCAPPP_ENABLE_PCAP_SET_DIRECTION=<ON/OFF>`**  | Enable set direction for capturing incoming or outgoing packets (default value is `OFF`)                                                                                                                                                                                    |
 
-## Build the code
+## Installation
 
-After running the config script, you're can safely build the code:
+When building for Android the build machine is usually different from the target device, so you may want to gather the lib and header files in a directory instead of installing it in the build machine.
```

Comment:
usually when cross-compiling, we use "host" and "target" to speak about the build machine and target device

Reply:
Fixed in https://github.com/PcapPlusPlus/pcapplusplus.github.io/pull/48/commits/cbb93cd88cf00bd702a3ec936eceba90f8bfc47f